ls1intum/Artemis
# `Communication`: Prevent senders from getting notifications of their own messages

### Checklist
#### General


#### Server

### Motivation and Context


Currently the sender of a message is sent a push notification about their own message.

### Description

With these changes the sender of a message will be filtered out. 

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Use the Android or iOS application
2. Enable push notifications and conversation notifications
3. Write a message in any conversation
4. Check if you have not received a push notification about your own message.

#### Code Review
#### Manual Tests
